File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.cpp
class Solution {
public:
    string firstPalindrome(vector<string>& words) {
         for(int i=0;i<words.size();i++){
            string temp=words[i];
            char dupli[1000];
            int k=0;
            for(int j=temp.length()-1;j>=0;j--){
               dupli[k]=temp[j];
               k++;
            }
            dupli[k] = '\0';
            if(temp==dupli){
                return temp;
            }
        }
        return "";
    }
};

File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.java
class Solution {
    public String firstPalindrome(String[] words) {
        for(int i=0;i<words.length;i++){
            String st=words[i];
            char[] copy = new char[st.length()];

            int k=0;
            for(int j=st.length()-1;j>=0;j--){
              copy[k]=st.charAt(j);
              k++;
            }
            String reversed=new String(copy);
            if(st.equals(reversed)){
                return reversed;
            }
        }
return "";
    }
}